docs/ai/quickstarts/chat-local-model.md

```diff
@@ -84,12 +84,12 @@ The Semantic Kernel SDK provides many services and features to connect to AI mod
 
     The preceding code accomplishes the following:
 
-    - Creates an `OllamaChatClient` that implements the `IChatClient` interface.
-        - This interface provides a loosely coupled abstraction you can use to chat with AI Models.
-        - You can later change the underlying chat client implementation to another model, such as Azure OpenAI, without changing any other code.
-    - Creates a `ChatHistory` object to store the messages between the user and the AI model.
-    - Retrieves a prompt from the user and stores it in the `ChatHistory`.
-    - Sends the chat data to the AI model to generate a response.
+    * Creates an `OllamaChatClient` that implements the `IChatClient` interface.
```

Comment:
Just curious here - don't we usually use hyphens for bullet points? I think either way is fine but standardizing on one would probably be best.

Reply:
I don't have a strong preference between asterisks and hyphens, but the linter likes consistency within a single file. And since asterisks were used first in this file, that's what the linter wanted them all to be.